### PR TITLE
[BPK-861] Add iOS platform tokens

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [x] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

**Comment by k0nserv** [comment]
Hmm we'll need colours too

File: changelog.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ _Nothing yet!_
 **Fixed:**
 - bpk-tokens:
   - Remove unused output formats for iOS and Android which aren't relevant when the platform is web.
+  - Added new outputs for the iOS platform in `bpk-token/tokens/ios/base.ios.json` and `bpk-tokens/tokens/react.native.es6.js`. Thes interface are not to be considered stable yet however and they may still change.
```
Comment (tvooo) [inline changelog.md]: "These interfaces"

File: packages/bpk-tokens/src/ios/base/spacing.json [hunk 2]
```diff
@@ -7,9 +7,6 @@
     "category": "spacings"
   },
   "props": {
-    "SPACING_XS": {
```
Comment (tvooo) [inline packages/bpk-tokens/src/ios/base/spacing.json]: Is there no spacing_xs on native?

File: packages/bpk-tokens/src/ios/base/typography.json [hunk 3]
```diff
@@ -78,145 +78,160 @@
...
-      "value": "{!FONT_WEIGHT_SEMIBOLD}",
+    "LETTER_SPACING_XL": {
+      "value": "{!LETTER_SPACING_XL}",
       "type": "number",
       "category": "typesettings"
     },
+    "LETTER_SPACING_XXL": {
+      "value": "{!LETTER_SPACING_XXL}",
+      "type": "number",
+      "category": "typesettings"
```
Comment (tvooo) [inline packages/bpk-tokens/src/ios/base/typography.json]: We'll probably have to overhaul these "category" fields at some point...

File: packages/bpk-tokens/src/ios/base/spacing.json (hunk 2 above)
Comment (k0nserv) [inline packages/bpk-tokens/src/ios/base/spacing.json]: Nope not per the craft library.

File: packages/bpk-tokens/src/ios/base/typography.json (hunk 3 above)
Comment (k0nserv) [inline packages/bpk-tokens/src/ios/base/typography.json]: Yeah quite possibly, I tried to align them with web as much as I could here

File: changelog.md (hunk 1 above)
Comment (k0nserv) [inline changelog.md]: Good catch

**Pull request merged**